README.md

```diff
@@ -51,13 +51,24 @@ rather than IP numbers in a Hyper-V environment. On CentOS systems,
 this requires the bind-utils package (updatedns requires the 'dig'
 program). The bind-utils package isn't otherwise necessary.
 
+- Install minimum version of Ruby (the system must have 1.9 or higher, but 2.2.6 is recommended)
```

Comment:
This is part of a list that's actually "per-system-type", but this requirement is for all systems. Suggest you change the wording to clarify. Perhaps remove from the list? Like change this line to something like:

```
Install Install minimum version of Ruby (the system must have 1.9 or higher,
but 2.2.6 is recommended). This is needed for all systems. This can be done
with the following commands:
```
Then have 55-64. And this isn't part of the per-system list.

This is a nit, but you get the idea.